ts/features/bonus/cgn/store/reducers/eyca/activation.ts

```diff
@@ -0,0 +1,73 @@
+// bonus reducer
+import { getType } from "typesafe-actions";
+import { createSelector } from "reselect";
+import { Action } from "../../../../../../store/actions/types";
+import { GlobalState } from "../../../../../../store/reducers/types";
+import {
+  cgnEycaActivationRequest,
+  cgnEycaActivationStatus
+} from "../../actions/eyca/activation";
+import { EycaActivationDetail } from "../../../../../../../definitions/cgn/EycaActivationDetail";
+
+export enum CgnEycaActivationProgressEnum {
+  "UNDEFINED" = "UNDEFINED",
```

Comment:
Will you use all these status?
At the moment the reducer uses only `UNDEFINED`, `PROGRESS`, `ERROR`

Reply:
I guess we can remove some of these statuses

may we move this discussion in the PR of networking/orchestration saga? #2855